assets/js/app.js

```diff
@@ -0,0 +1,149 @@
+
+// collects the data of the form
+function collectData() {
+
+    const url = document.getElementById(`url`).value
+    const title = document.getElementById(`title`).value
+    const description = document.getElementById(`description`).value
+
+    return {
+        id: Date.now(),
```

Comment:
Date.now() is not the best option for an `id`. You can easily get identical IDs.